### PR TITLE
Fix #36159 retro-bit.com

### LINKED ISSUE #36159: retro-bit.com
`http://retro-bit.com/`

Newsletter overlay annoyance.

Adguard Base + Annoyances + Portuguese + Russian + Social + Spyware
uBlock Origin 1.20.0
Firefox 67

### PR DESCRIPTION
**Pull request by tvinzz**
https://github.com/AdguardTeam/AdguardFilters/issues/36159

### CODE REVIEW

File: AnnoyancesFilter/sections/popups.txt [hunk 1]
```diff
@@ -80,6 +80,7 @@
 !
 ! Specific sites
 !
+retro-bit.com#%#document.cookie = "empopup=null";
```
Comment (Alex-302) [inline AnnoyancesFilter/sections/popups.txt]: better use CSS rules

Comment (tvinzz) [reply AnnoyancesFilter/sections/popups.txt]: done with two CSS rules

File: AnnoyancesFilter/sections/popups.txt [hunk 2]
```diff
@@ -80,7 +80,8 @@
 !
 ! Specific sites
 !
-retro-bit.com#%#document.cookie = "empopup=null";
+retro-bit.com##.modals-wrapper.popup-newsletter
```
Comment (BlazDT) [inline AnnoyancesFilter/sections/popups.txt]: Must be hidden by #$# rule or else user will not have scrolling when using a product which does not support #$# rules.

**Review by BlazDT — CHANGES_REQUESTED** [review]

**Review by BlazDT — APPROVED** [review]

File: AnnoyancesFilter/sections/popups.txt (hunk 2 above)
Comment (Alex-302) [reply AnnoyancesFilter/sections/popups.txt]: and `.modals-wrapper` is unnecessary

File: AnnoyancesFilter/sections/popups.txt [hunk 3]
```diff
@@ -80,6 +80,8 @@
 !
 ! Specific sites
 !
+retro-bit.com#$#.modals-wrapper.popup-newsletter { display: none !important }
+retro-bit.com#$#body._has-modal { overflow: visible !important }
```
Comment (Alex-302) [inline AnnoyancesFilter/sections/popups.txt]: `._has-modal` redundant part

File: AnnoyancesFilter/sections/popups.txt [hunk 4]
```diff
@@ -80,6 +80,8 @@
 !
 ! Specific sites
 !
+retro-bit.com#$#.modals-wrapper.popup-newsletter { display: none !important }
```
Comment (Alex-302) [inline AnnoyancesFilter/sections/popups.txt]: `.modals-wrapper` part is redundant

**Review by Alex-302 — CHANGES_REQUESTED** [review]

**Review by Alex-302 — APPROVED** [review]

**Review by Alex-302 — APPROVED** [review]

**Pull request merged**